step3/app.js

```diff
@@ -0,0 +1,77 @@
+const express = require('express');
+const fs = require('fs');
+const app = express();
+
+PORT = 3000;
+
+const getItems = () => {
+    const getItems = JSON.parse(fs.readFileSync('./data.json','utf-8'));
+    return getItems;
+}
+
+const save = (items) => {
+    fs.writeFileSync('./data.json',JSON.stringify(items),'utf-8');
+}
```

Comment:
```suggestion
const save = (items) => fs.writeFileSync('./data.json',JSON.stringify(items),'utf-8');
```
이 부분도 이렇게 표현할 수 있답니다!

Reply:
그리고 이렇게 파일을 관리하는 함수를 module로 분리해서 관리해주면 어떨까요?